Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -23,9 +23,17 @@ $ strings <yourapp> | grep "myURLscheme://"
 
 #### Dynamic Analysis
 
-Once you know, what an URL structure is, you should try fuzzing an URL to force an application to perform some malicious action.
+As custom URL schemes may contain bugs, it is worth fuzzing those URLs. To do it you may use [IDB](http://www.idbtool.com/) tool:
 
--- TODO [Add instruction of using dynamic/ipc/open_uri: Test IPC attacks by launching URI Handlers in Needle: https://labs.mwrinfosecurity.com/blog/needle-how-to/] --
+- Connect IDB tool with your device and select tested application. [Here](http://www.idbtool.com/documentation/setup.html) you can find a detailed guide in setting it up.
+- Go to `URL Handlers` section. In `URL schemes` click `Refresh` button and you will find on the left side list of all custom schemes defined in tested application. You can load those schemes using `Open` button on the right side. By simply opening blank URI scheme you may discover hidden functionality (e.g. debug window) or bypass local authentication.
```

Comment:
> you will find on the left side list of all custom schemes defined in tested application.

Should this be "a list"?

> By simply opening blank URI scheme you may discover hidden functionality (e.g. debug window) or bypass local authentication.

I don't understand this part. What is the blank URI scheme? How does it bind hidden functionality? Maybe you can describe how this works here.